deker_server_adapters/utils.py

```diff
@@ -0,0 +1,41 @@
+import traceback
+
+from logging import getLogger
+from random import randint
+from typing import Dict, Optional, Sequence
+
+from httpx import Client, Response
+
+from deker_server_adapters.consts import STATUS_OK
+
+
+logger = getLogger(__name__)
+
+
+def request_in_cluster(
```

Comment:
Since it is universal solution for cluster and non-cluster servers
I would change the name to more universal
Maybe it is worth to add fast path for the case of only 1 node in cluster(non-cluster) because randint could be slow